pywinauto/uia_defines.py

```diff
@@ -42,7 +106,8 @@
 # header: UIAutomationClient.h
 
 def _build_pattern_ids_dic():
-    """A helper procedure to build a registry of control patterns 
+    """
```

Comment:
I'm just trying to reduce number of landscape.io warnings. I'd suggest to follow both QC and Landscape recommendations. Landscape says that multi line comment should have 1 line below quotes, 1 blank line and then more detailed description. Closing quotes should stay on separate line as well. For one line doc string every thing is at one line.